scripts/runSchoolApi.js

```diff
@@ -0,0 +1,115 @@
+'use strict'
+
+const axios = require('axios')
+
+const path = '/admin/api/v1'
+const endPointSchools = '/admin/schools'
+const endPointUsers = '/admin/users'
+const federalStateNames = {
+	nbc: 'Niedersachsen',
+	brb: 'Brandenburg',
+}
+const users = {
+	admin1_nbc: 'administrator',
+	teacher1_nbc: 'teacher',
+	student1_nbc: 'student',
+}
+
+const headers = {
+	'Content-Type': 'application/json',
+}
+
+const extractSubdomain = url => {
+	try {
+		const parsedUrl = new URL(url)
+		return parsedUrl.hostname.split('.')[1]
+	} catch (error) {
+		console.error('Error parsing URL:', error.message)
+		return null
+	}
+}
+
+const createSchool = async (schoolUrl, headers) => {
+	try {
+		const payload = {
+			name: `cypress-automated-tests`,
+			federalStateName: federalStateNames[extractSubdomain(schoolUrl)],
+		}
+		const response = await axios.post(schoolUrl, payload, { headers })
+		const { id } = response.data
+		console.log('School created with ID')
```

Comment:
Should we add the actual `id` to the log like `console.log('School created with ID', id)` or is that a security risk?

Reply:
Done